README.md

```diff
@@ -41,6 +42,10 @@ The `FILENAME` variable sets the filename that is used. The default value is
 information). To dump on multiple interfaces simply add more interfaces to this 
 variable seperated by a whitespace (e.g. "eth0 eth1").
 
+The `FORMAT` variable sets the format of the written file. Use the variable `dump.pcap` as
+the `FILENAME` option and `pcap` for `FILENAME` to use the common pcap-format. Other formats are described in
+the [official documentation](https://www.wireshark.org/docs/man-pages/tshark.html).
+
```

Comment:
this section here is rather confusing. why is `dump.pcap` a variable? isnt it the value of a variable?
i shall use `dump.pcap` for `FILENAME` and also `pcap` for `FILENAME`?

in generell: why dont you just use something like `ADDITIONAL_FLAGS` to transport any flag someone likes down to `dumpcap` and let users use "-F pcap" if they desire it. document `how to change from pcapng to pcap format` here for sure, but don't make the script more complex then it needs to be

Reply:
>i shall use dump.pcap for FILENAME and also pcap for FILENAME?
Yes, using the default FILENAME `dump` and using the value `pcap` as the `FORMAT` will not result in .pcap files. The FILENAME also has to be renamed to `dump.pcap` making this a bit confusing to explain

Reply:
Yes, ok. But you declare two times to set `FILENAME` -- the second time you probably refer to `FORMAT`.